data/data/manifests/openshift/baremetal-provisioning-config.yaml.template

```diff
@@ -10,3 +10,4 @@ spec:
   provisioningDHCPRange: "{{.Baremetal.ProvisioningDHCPRange}}"
   provisioningOSDownloadURL: "{{.ProvisioningOSDownloadURL}}"
   watchAllNamespaces: false
+  disableVirtualMediaTLS: "{{ .Baremetal.DisableVirtualMediaTLS }}"
```

Comment:
This should not be here. We will use the absence of this field by default as a sign to keep the default behavior.